pages/api/Schema/index.ts

```diff
@@ -0,0 +1,15 @@
+import { buildFederatedSchema } from '@apollo/federation';
+import ContactFiltersResolvers from './ContactFilters/resolvers';
+import ContactFiltersTypeDefs from './ContactFilters/typeDefs';
+import ExportContactsResolvers from './ExportContacts/resolvers';
+import ExportContactsTypeDefs from './ExportContacts/typeDefs';
+import TaskAnalyticsResolvers from './TaskAnalytics/resolvers';
+import TaskAnalyticsTypeDefs from './TaskAnalytics/typeDefs';
+
+const schema = buildFederatedSchema([
```

Comment:
I thought it would be a good idea to have this in one place that we can just export.